grammars/php.cson

```diff
@@ -3493,7 +3500,7 @@
         'captures':
           '1':
             'name': 'punctuation.definition.variable.php'
-        'match': '(?i)(\\$+)[a-z_\\x{7f}-\\x{ff}][a-z0-9_\\x{7f}-\\x{ff}]*\\b'
+        'match': '(?i)(\\$+)[a-z_\\x{7f}-\\x{7fffffff}][a-z0-9_\\x{7f}-\\x{7fffffff}]*\\b'
```

Comment:
Word boundary needs to be removed from here as a lot of unicode code points are not considered as word characters, simplest case are variables such as `$🐘` This might be also needed to be done elsewhere, I'll take a look at the other changed lines later.

Since we are pretty much matching all of the characters in existence, except for the few in ASCII range, there is no need for a boundary check as it terminates at the characters not specified here.